### PR TITLE
Zenodo: Add meta-data

### PR DESCRIPTION
**Pull request by ax3l**
Add creators, contributors and general meta-data for zenodo source code archives.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
```
Comment (ax3l) [inline .zenodo.json]: @jlvay what is your ORCiD? :)
```suggestion
        "name": "Vay, Jean-Luc",
        "orcid": "0000-0002-0040-799X"
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Almgren, Ann"
```
Comment (ax3l) [inline .zenodo.json]: @asalmgren what is your ORCiD? :)
```suggestion
        "name": "Almgren, Ann",
        "orcid": "0000-0003-2103-312X"
```

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Amorim, L\u00edgia Diana"
```
Comment (ax3l) [inline .zenodo.json]: @LDAmorim what is your ORCiD? :)
```suggestion
        "name": "Amorim, L\u00edgia Diana",
        "orcid": "..."
```

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Laboratoire d'Optique Appliqu\u00e9e",
+        "name": "Andriyash, Igor"
```
Comment (ax3l) [inline .zenodo.json]: @hightower8083 what is your ORCiD? :)
```suggestion
        "name": "Andriyash, Igor",
        "orcid": "0000-0003-0313-4496"
```

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Belkin, Daniel"
```
Comment (ax3l) [inline .zenodo.json]: @danielbelkin what is your ORCiD? :)
```suggestion
        "name": "Belkin, Daniel",
        "orcid": "0000-0002-3936-6419"
```

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Bizzozero, David"
```
Comment (ax3l) [inline .zenodo.json:30]: @dbizzozero what is your ORCiD? :)
```suggestion
        "name": "Bizzozero, David",
        "orcid": "..."
```

File: .zenodo.json [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Blelly, Aurore"
```
Comment (ax3l) [inline .zenodo.json:34]: @ablelly what is your ORCiD? :)
```suggestion
        "name": "Blelly, Aurore",
        "orcid": "..."
```

File: .zenodo.json [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "LIDYL, CEA-Universit\u00e9 Paris-Saclay, CEA Saclay",
+        "name": "Fedeli, Luca"
```
Comment (ax3l) [inline .zenodo.json]: @lucafedeli88 what is your ORCiD? :)
```suggestion
        "name": "Fedeli, Luca",
        "orcid": "0000-0002-7215-4178"
```

File: .zenodo.json [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "SLAC National Accelerator Laboratory",
+        "name": "Ge, Lixin"
```
Comment (ax3l) [inline .zenodo.json]: @lge0303 what is your ORCiD? :)
```suggestion
        "name": "Ge, Lixin",
        "orcid": "0000-0002-1442-0706"
```

File: .zenodo.json [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Gott, Kevin"
```
Comment (ax3l) [inline .zenodo.json]: @kngott what is your ORCiD? :)
```suggestion
        "name": "Gott, Kevin",
        "orcid": "0000-0003-3244-5525"
```

File: .zenodo.json [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Harrison, Cyrus"
```
Comment (ax3l) [inline .zenodo.json]: @cyrush what is your ORCiD? :)
```suggestion
        "name": "Harrison, Cyrus",
        "orcid": "0000-0001-8985-0906"
```

File: .zenodo.json [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Huebl, Axel",
+        "orcid": "0000-0003-1943-7141"
+      },
+      {
+        "affiliation": "Lawrence Livermore National Laboratory",
+        "name": "Grote, David"
```
Comment (ax3l) [inline .zenodo.json]: @dpgrote what is your ORCiD? :)
```suggestion
        "name": "Grote, David",
        "orcid": "0000-0002-4057-8582"
```

File: .zenodo.json [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Gu, Junmin",
+        "orcid": "0000-0002-1521-8534"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Jambunathan, Revathi"
```
Comment (ax3l) [inline .zenodo.json]: @RevathiJambunathan what is your ORCiD? :)
```suggestion
        "name": "Jambunathan, Revathi",
        "orcid": "0000-0001-9432-2091"
```

File: .zenodo.json [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory, now DESY",
+        "name": "Th\u00e9venet, Maxence"
```
Comment (ax3l) [inline .zenodo.json]: @MaxThevenet what is your ORCiD? :)
```suggestion
        "name": "Th\u00e9venet, Maxence",
        "orcid": "0000-0001-7216-2277"
```

File: .zenodo.json [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Richardson, Glenn"
```
Comment (ax3l) [inline .zenodo.json]: @gtrichardson what is your ORCiD? :)
```suggestion
        "name": "Richardson, Glenn",
        "orcid": "0000-0003-0313-4496"
```

File: .zenodo.json [hunk 16]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Shapoval, Olga"
```
Comment (ax3l) [inline .zenodo.json]: @oshapoval :)
```suggestion
        "name": "Shapoval, Olga",
        "orcid": "0000-0003-4003-4507"
```

File: .zenodo.json [hunk 17]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Lehe, R\u00e9mi"
```
Comment (ax3l) [inline .zenodo.json]: @RemiLehe :)
```suggestion
        "name": "Lehe, Remi",
        "orcid": "0000-0002-3656-9659"
```

File: .zenodo.json [hunk 18]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Loring, Burlen"
```
Comment (ax3l) [inline .zenodo.json]: @burlen :)
```suggestion
        "name": "Loring, Burlen",
        "orcid": "0000-0002-4678-8142"
```

File: .zenodo.json [hunk 19]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Myers, Andrew"
```
Comment (ax3l) [inline .zenodo.json]: @atmyers :)
```suggestion
        "name": "Myers, Andrew",
        "orcid": "0000-0001-8427-8330"
```

File: .zenodo.json [hunk 20]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Rowan, Michael E."
```
Comment (ax3l) [inline .zenodo.json]: @mrowan137 :)
```suggestion
        "name": "Rowan, Michael E.",
        "orcid": "0000-0003-2406-1273"
```

File: .zenodo.json [hunk 21]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Yang, Eloise"
```
Comment (ax3l) [inline .zenodo.json]: @eloisejyang :)
```suggestion
        "name": "Yang, Eloise",
        "orcid": "0000-0002-9319-4216"
```

File: .zenodo.json [hunk 22]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zhang, Weiqun"
```
Comment (ax3l) [inline .zenodo.json]: @WeiqunZhang :)
```suggestion
        "name": "Zhang, Weiqun",
       "orcid": "0000-0001-8092-1974"
```

File: .zenodo.json [hunk 23]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zhao, Yinjian"
```
Comment (ax3l) [inline .zenodo.json]: @Yin-YinjianZhao :)
```suggestion
        "name": "Zhao, Yinjian",
        "orcid": "0000-0003-4362-3630"
```

File: .zenodo.json [hunk 24]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zoni, Edoardo"
```
Comment (ax3l) [inline .zenodo.json]: @EZoni :)
```suggestion
        "name": "Zoni, Edoardo",
        "orcid": "0000-0001-5662-4646"
```

File: .zenodo.json [hunk 25]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "LIDYL, CEA-Universit\u00e9 Paris-Saclay, CEA Saclay",
+        "name": "Zaim, Ne\u00efl"
```
Comment (ax3l) [inline .zenodo.json]: @NeilZaim :)
```suggestion
        "name": "Zaim, Ne\u00efl",
        "orcid": "0000-0003-0313-4496"
```

File: .zenodo.json [hunk 26]
```diff
@@ -0,0 +1,162 @@
...
+      }
+    ],
+    "contributors": [
+      {
+        "affiliation": "Intel",
+        "type": "Other",
+        "name": "Cohn, Robert"
```
Comment (ax3l) [inline .zenodo.json:147]: @rscohn2 what is your ORCiD? :)
```suggestion
        "name": "Cohn, Robert",
        "orcid": "..."
```

File: .zenodo.json [hunk 27]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Bloomberg LP",
+        "type": "Other",
+        "name": "Levine, Michael"
+      },
+      {
+        "type": "Other",
+        "name": "Liu, Bin"
```
Comment (ax3l) [inline .zenodo.json]: @steelan what is your affiliation & ORCiD? :)
```suggestion
        "affiliation": "Helmholtz Institute Jena",
        "name": "Liu, Bin",
        "orcid": "0000-0002-8066-184X"
```

File: .zenodo.json [hunk 28]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "type": "Other",
+        "name": "nckwcq"
```
Comment (ax3l) [inline .zenodo.json]: @nckwcq what is your real name, affiliation and ORCiD? :)
```suggestion
        "affiliation": "SLAC National Accelerator Laboratory",
        "name": "Ng, Cho-Kuen",
        "orcid": "0000-0002-1903-904X"
```

File: .zenodo.json [hunk 29]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "type": "Other",
+        "name": "Park, Jaehong"
```
Comment (ax3l) [inline .zenodo.json:169]: @jaehong2010 what is your ORCiD? :)
```suggestion
        "name": "Park, Jaehong",
        "orcid": "..."
```

File: .zenodo.json [hunk 30]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "type": "Other",
```
Comment (ax3l) [inline .zenodo.json:140]: I am not sure which GitHub user that is or his ORCiD, but he was picked but by zenodo.

File: .zenodo.json (hunk 20 above)
Comment (mrowan137) [reply .zenodo.json]: 0000-0003-2406-1273

File: .zenodo.json (hunk 12 above)
Comment (dpgrote) [reply .zenodo.json]: 0000-0002-4057-8582

File: .zenodo.json (hunk 24 above)
Comment (EZoni) [reply .zenodo.json]: Thanks Axel, here it is: 0000-0001-5662-4646

File: .zenodo.json [hunk 31]
```diff
@@ -0,0 +1,163 @@
...
+        "name": "Rowan, Michael E."
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Yang, Eloise"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Zhang, Weiqun",
+       "orcid": "0000-0001-8092-1974"
```
Comment (ax3l) [inline .zenodo.json]: I did a formatting upsi
```suggestion
        "orcid": "0000-0001-8092-1974"
```

File: .zenodo.json [hunk 32]
```diff
@@ -0,0 +1,164 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Rowan, Michael E."
+      },
+      ,
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Weber, Gunther H.",
+        "orcid": "..."
```
Comment (ax3l) [inline .zenodo.json]: @ghweber what is your ORCiD? :)
```suggestion
        "orcid": "0000-0002-1794-1398"
```

Comment (ghweber) [reply .zenodo.json]: 0000-0002-1794-1398

Comment (ghweber) [reply .zenodo.json]: Full URL https://orcid.org/0000-0002-1794-1398

File: .zenodo.json (hunk 10 above)
Comment (kngott) [reply .zenodo.json]: 0000-0003-3244-5525

File: .zenodo.json (hunk 5 above)
Comment (danielbelkin) [reply .zenodo.json]: https://orcid.org/0000-0002-3936-6419

File: .zenodo.json (hunk 25 above)
Comment (NeilZaim) [reply .zenodo.json]: https://orcid.org/0000-0002-9582-5894

Thanks :)

File: .zenodo.json (hunk 14 above)
Comment (MaxThevenet) [reply .zenodo.json]: https://orcid.org/0000-0001-7216-2277, I just updated your suggestion above. Thanks for taking care of this!

File: .zenodo.json (hunk 3 above)
Comment (LDAmorim) [reply .zenodo.json]:
```suggestion
        "name": "Amorim, L\u00edgia Diana",
        "orcid": "0000-0002-1445-0032"
```

Comment (LDAmorim) [reply .zenodo.json]: Thanks @ax3l !

File: .zenodo.json (hunk 27 above)
Comment (steelan) [reply .zenodo.json]: affiliation:   Helmholtz Institute Jena
orcid:   0000-0002-8066-184X

Thanks :)

File: .zenodo.json (hunk 4 above)
Comment (hightower8083) [reply .zenodo.json]: @ax3l Thanx for including me to this honorable list =) ! 
Shamefully, I've barely ever used ORCiD, but  here is one: 0000-0003-0313-4496

File: .zenodo.json (hunk 15 above)
Comment (gtrichardson) [reply .zenodo.json]: 0000-0001-9353-2791 (https://orcid.org/0000-0001-9353-2791). Newly made, just for you!

**Comment by nckwcq** [comment]
https://orcid.org/0000-0002-1903-904X

Thanks,
Cho

**Comment by burlen** [comment]
@ax3l 
0000-0002-4678-8142
https://orcid.org/0000-0002-4678-8142

File: .zenodo.json (hunk 19 above)
Comment (atmyers) [reply .zenodo.json]: orcid=0000-0001-8427-8330

File: .zenodo.json (hunk 6 above)
Comment (ax3l) [reply .zenodo.json:30]: ping @dbizzozero what is your ORCiD? :)

File: .zenodo.json (hunk 7 above)
Comment (ax3l) [reply .zenodo.json:34]: ping @ablelly what is your ORCiD? :)

File: .zenodo.json (hunk 8 above)
Comment (ax3l) [reply .zenodo.json]: ping @lucafedeli88 what is your ORCiD? :)

File: .zenodo.json (hunk 9 above)
Comment (ax3l) [reply .zenodo.json]: ping @lge0303 what is your ORCiD? :)

File: .zenodo.json (hunk 11 above)
Comment (ax3l) [reply .zenodo.json]: ping @cyrush what is your ORCiD? :)

File: .zenodo.json (hunk 13 above)
Comment (ax3l) [reply .zenodo.json]: ping @RevathiJambunathan what is your ORCiD? :)

File: .zenodo.json (hunk 16 above)
Comment (ax3l) [reply .zenodo.json]: ping @oshapoval what is your ORCiD? :)

File: .zenodo.json (hunk 21 above)
Comment (ax3l) [reply .zenodo.json]: ping @eloisejyang what is your ORCiD? :)

File: .zenodo.json (hunk 26 above)
Comment (ax3l) [reply .zenodo.json:147]: ping @rscohn2 what is your ORCiD? :)

File: .zenodo.json (hunk 29 above)
Comment (ax3l) [reply .zenodo.json:169]: ping @jaehong2010 what is your ORCiD? :)

File: .zenodo.json (hunk 6 above)
Comment (ax3l) [reply .zenodo.json:30]: Offline asked: has no ORCiD

File: .zenodo.json (hunk 7 above)
Comment (ax3l) [reply .zenodo.json:34]: Maybe https://orcid.org/0000-0001-7942-310X?

File: .zenodo.json (hunk 13 above)
Comment (RevathiJambunathan) [reply .zenodo.json]: Thank you for the ping @ax3l . Just updated it

File: .zenodo.json (hunk 9 above)
Comment (lge0303) [reply .zenodo.json]: Here is mine:
https://orcid.org/0000-0002-1442-0706

Thanks,

Lixin

File: .zenodo.json (hunk 16 above)
Comment (oshapoval) [reply .zenodo.json]: Thanks for the ping @ax3l !
https://orcid.org/0000-0003-4003-4507

File: .zenodo.json (hunk 8 above)
Comment (lucafedeli88) [reply .zenodo.json]: I am sorry! Here it is: https://orcid.org/0000-0002-7215-4178

File: .zenodo.json (hunk 11 above)
Comment (cyrush) [reply .zenodo.json]: https://orcid.org/0000-0001-8985-0906

**Comment by ax3l** [comment]
Ok, I think we collected a reasonable amount of ORCiDs. Later updates always welcome via PRs :)

Thank you all for your contributions to WarpX! :rocket: :sparkles:

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**